pkg/event/handler.go

```diff
@@ -1,19 +1,48 @@
 package event
 
+import "sort"
+
+var (
+	handlers    = make(map[Source]Handler)
+	orderOrigin = make(map[int][]Handler)
```

Comment:
what's this?

Reply:
you may see the function `Register` at the end, the `Register` function will be call at event handler `init` function.
`Register` will filled in infos into above variables.